efel/units/units.json

```diff
@@ -0,0 +1,129 @@
+{
+    "ADP_peak_amplitude": "mV",
+    "ADP_peak_values": "mV",
+    "AHP_depth": "mV",
+    "AHP_depth_abs": "mV",
+    "AHP_depth_abs_slow": "mV",
+    "AHP_depth_diff": "mV",
+    "AHP_depth_from_peak": "mV",
+    "AHP_depth_last": "mV",
+    "AHP_depth_slow": "mV",
+    "AHP_slow_time": "constant",
+    "AHP_time_from_peak": "ms",
+    "AHP_time_from_peak_last": "mV",
```

Comment:
A time in mV ?

Reply:
Very good catch, that's why these should be tested. I parsed them programmatically, the error originates here: https://efel.readthedocs.io/en/latest/eFeatures.html#libv5-ahp-time-from-peak-last

Reply:
I'm fixing that one, please double check for more inconsistencies in the features.

Reply:
Fixed in both docs and API.